lib/ncr/mailboxes.rb

```diff
@@ -0,0 +1,29 @@
+module Ncr
+  module Mailboxes
+    def self.ba61_tier1_budget
+      self.approver_with_role("BA61_tier1_budget_approver")
+    end
+
+    def self.ba61_tier2_budget
+      self.approver_with_role("BA61_tier2_budget_approver")
+    end
+
+    def self.ba80_budget
+      self.approver_with_role("BA80_budget_approver")
+    end
+
+    def self.ool_ba80_budget
+      self.approver_with_role("OOL_BA80_budget_approver")
+    end
+
+    protected
+
+    def self.approver_with_role(role_name)
```

Comment:
> protected class methods are tricky

Not sure that I fully grok the "virtual class" class thing, but is it really a problem? Especially since Ruby's `protected`/`private` behavior is...unique...I just use them to indicate to other developers "you probably don't want to call this from outside the method/module.

> what about making the methods in here instance methods

I personally think a singleton makes sense here, since conceptually there's no reason to create multiple `Ncr::Mailboxes` instances. Sound reasonable?